docs/whichkey/usage.md

```diff
@@ -193,6 +193,25 @@ An example of a `settings.json` file that overrides the space menu is as follows
 The default value can be found in the `contributes.configuration.whichkey.bindings.default` section of the `package.json` in this repo.
 You can use the default value as an example to craft your own custom menu.
 
+### Multiple Menus
+
+Only a single global menu is supported natively. 
```

Comment:
That's not entirely correct, see https://vspacecode.github.io/docs/whichkey/reference#show. One can pass an array of the bindings directly.